public/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hello, Rachel</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
Ah, careful what version of Bootstrap you're using! The most recent version is 4.4. 3.4.1 is pretty old.

It being old isn't a problem _per se_, but keep it in mind as you're reading documentation. If you're reading something for 4.4, it might not work with 3.4 (e.g., different class names).